install-dependencies.sh

```
@@ -0,0 +1,121 @@
+    else
+        echo -e "${RED}❌ Failed to install Angular CLI${NC}"
+        exit 1
+    fi
+fi
+
+# Install Backend Dependencies
+echo -e "\n${YELLOW}🔧 Installing Backend Dependencies...${NC}"
+echo -e "${BLUE}📂 Navigating to backend directory...${NC}"
+
+if [ -d "backend" ]; then
+    cd backend
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)